src/topicmodeller/topicmodeller/topicmodeller.py

```diff
@@ -170,7 +192,9 @@ def classify_and_dump_json(documents_folder, tm_data_folder, output_folder):
         (scraper_document, document) = _json_sd_file_to_sd_and_tmd(os.path.join(documents_folder, file_name))
 
         # pylint: disable=E1101
-        json = jsonpickle.encode(TopicModellerDocument(scraper_document.link_element.url, topicmodeller.classify(document)))
+        json = jsonpickle.encode(TopicModellerDocument(scraper_document.link_element.origin_info.title,
+                                                       scraper_document.link_element.url,
+                                                       topicmodeller.classify(document)))
         # pylint: enable=E1101
         filename = os.path.join(output_folder, str(date) + '_' + str(i) + '.json')
```

Comment:
What is the name not found by pylint ? it may be solved by adding something to the pylint sys.path in pylintrc ?